pages/linux/sic.md

```diff
@@ -0,0 +1,29 @@
+# sic
+
+> simple irc client.
+> Part of the suckless.org tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to host (default: irc.oftc.net) with nickname (default: $USER):
+
+`sic -n {{nickname}}`
```

Comment:
I'm thinking this (from https://github.com/tldr-pages/tldr/pull/5694/files#r607888085)
```suggestion
- Connect to a host with the specified nickname:

`sic -h {{irc.ofct.net}} -n {{$USER}}`
```

Reply:
seems legit